package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Use this to save yourself some time when working on a webpack configuration.",
   "main": "index.js",
   "scripts": {
-    "cover": "nyc --reporter=lcov --reporter=text npm t",
+    "cover": "nyc --reporter=lcov --reporter=text ava --coverage",
```

Comment:
You might also try: 

```
"cover": "nyc --reporter=lcov --reporter=text npm t --coverage"
```

Just to try more stuff out.

Thanks for your work on this :+1: